application/src/main/java/run/halo/app/notification/endpoint/UserNotificationEndpoint.java

```diff
@@ -106,9 +106,33 @@ Supplier<RouterFunction<ServerResponse>> userspaceScopedApis() {
                     )
                     .response(responseBuilder().implementationArray(String.class))
             )
+            .DELETE("/notifications/{name}", this::deleteNotification,
```

Comment:
Please feel free to add the corresponding e2e testing case for the new API. See also https://github.com/halo-dev/halo/pull/4920/files#diff-f576994e2631a97d58544de7da11ec699440776f560913a49035a50bf6f5b361R138

Reply:
Thank you for reminding me